### PR TITLE
fix: run with docker

### LINKED ISSUE #15: Error MODULE_NOT_FOUND when running with docker
Standalone
image

Docker compose
image

```
frontend-1  | node:internal/modules/cjs/loader:1251
frontend-1  |   throw err;
frontend-1  |   ^
frontend-1  | 
frontend-1  | Error: Cannot find module '/app/server.js'
frontend-1  |     at Module._resolveFilename (node:internal/modules/cjs/loader:1248:15)
frontend-1  |     at Module._load (node:internal/modules/cjs/loader:1074:27)
frontend-1  |     at TracingChannel.traceSync (node:diagnostics_channel:315:14)
frontend-1  |     at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
frontend-1  |     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:166:5)
frontend-1  |     at node:internal/main/run_main_module:30:49 {
frontend-1  |   code: 'MODULE_NOT_FOUND',
frontend-1  |   requireStack: []
frontend-1  | }
frontend-1  | 
frontend-1  | Node.js v22.9.0

```

### PR DESCRIPTION
**Pull request by HazimAr**
Before it was using the `littlehorse-enterprises/alpine-nginx-nodejs` as a base image
The problem is that it runs `node /app/server.js` but because this is a monorepo we need to run `node /app/ui/server.js`

I also moved the docker stuff to the root of the project as it uses both folders when building.

closes #15

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -2,44 +2,44 @@
...
 fi
 
-if [ -z "${LHUT_TENANT_ID}" ]; then
+if [ ! "${LHUT_TENANT_ID+x}" ]; then
     echo "Provide the LHUT_TENANT_ID env variable"
     exit 1
 fi
 
-/entrypoint.sh
+npm run start -ws --if-present -- --port ${PORT} --hostname ${HOSTNAME}
```
Comment (sauljabin) [inline entrypoint.sh:45]: I'm not sure about this, this is a production build https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile#L70

npm run is not I guess

any idea @mijailr

**Review by sauljabin — CHANGES_REQUESTED** [review]
Let's discuss the Dockerfile

File: entrypoint.sh (hunk 1 above)
Comment (HazimAr) [reply entrypoint.sh:45]: you are correct @sauljabin I will change it

Comment (HazimAr) [reply entrypoint.sh]: > I'm not sure about this, this is a production build https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile#L70
> 
> npm run is not I guess
> 
> any idea @mijailr

Just FYI npm run start does `next start` which is production and essentially the same as `node server.js`

**Review by sauljabin — APPROVED** [review]
done, now it's working nice

**Pull request merged**